druid/examples/game_of_life.rs

```diff
@@ -143,16 +53,15 @@ impl Grid {
             for col in 0..GRID_SIZE {
                 let pos = GridPos { row, col };
                 let n_lives_around = self.n_neighbors(pos);
-                let life = self[pos];
-                // death by loneliness or overcrowding
-                if life && (n_lives_around < 2 || n_lives_around > 3) {
-                    indices_to_mutate.push(pos);
-                    continue;
-                }
-                // resurrection by life support
-                if !life && n_lives_around == 3 {
-                    indices_to_mutate.push(pos);
-                }
+                match (self[pos], n_lives_around) {
+                    // death by overcrowding
+                    (true, x) if x > 3 => indices_to_mutate.push(pos),
+                    // death by loneliness
+                    (true, x) if x < 2 => indices_to_mutate.push(pos),
+                    // resurrection by life support
+                    (false, 3) => indices_to_mutate.push(pos),
+                    (_, _) => {}
```

Comment:
nittiest nit: the idiomatic way to write this would be as `_ => (),`. The rationale is that `()` (the [`unit` type](https://doc.rust-lang.org/std/primitive.unit.html) is what is returned by functions & expressions with no explicit return type, and we have to return the same type as the other branches of the `match`, so we should just do that explicitly.

The difference is that `{}` is an empty *expression*, which happens to evaluate to `()`; so it works fine either way, but the first is more explicit.
```suggestion
                    _ => (),
```

Reply:
no problem, I missed that you could just do `_`. rust is such a big language its easy to miss some things.
Althought I thought `{}` was always considered idiomatic? 
[the rust reference](https://doc.rust-lang.org/reference/expressions/match-expr.html) uses it too.